### PR TITLE
Add minerva and minerva-update ansible tags

### PR DESCRIPTION
**Pull request by mgrauer**
@kotfic PTAL.

Let me know if I'm stomping on something or violating your design ideas.

### CODE REVIEW

File: ansible/site.yml [hunk 1]
```diff
@@ -39,7 +39,7 @@
       sudo: yes
 
     - role: minerva
-      minerva_version: "master"
+      minerva_version: "dataset_uploader"
```
Comment (aashish24) [inline ansible/site.yml]: @mgrauer why change this?

**Comment by mgrauer** [comment]
@aashish24 

Good catch, that was an accidental include from the branch I was testing.

File: Vagrantfile [hunk 2]
```diff
@@ -59,12 +59,15 @@ Vagrant.configure(2) do |config|
...
-
+  ansible_tags = ENV["MINERVA_VAGRANT_ANSIBLE_TAGS"] || false
 
   config.vm.provision "ansible" do |ansible|
     ansible.verbose = "vvvv"
 
+    if ansible_tags
+        ansible.tags = "minerva-update"
+    end
+
```
Comment (kotfic) [inline Vagrantfile]: Can we make this more general?  i.e.,  have MINERVA_VAGRANT_ANSIBLE_TAGS expect a comma separated list of tags that get passed directly to ansible.tags?

Comment (mgrauer) [inline Vagrantfile]: Thanks.  I had that in mind and left it out.

**Comment by mgrauer** [comment]
@kotfic 

fixed

**Comment by mgrauer** [comment]
@kotfic All issues we found today are addressed.  

PTAL at your convenience--this branch will keep.

**Comment by kotfic** [comment]
@mgrauer just tested this,  LGTM

**Pull request merged**